kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/startAdminServer.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
We should be using off-line WLST - not on-line WLST to create the domain. No need to start an admin server.

Reply:
Not sure why this is here - the WLST script is already off-line WLST.

Reply:
This script can and should be removed.